### ISSUE TITLE
Menyoo won't open on pressing F8

### ISSUE THREAD

**Issue by ArjanSinghSuri** [MAFINS/MenyooSP#617]
Hi. I have a Steam edition of GTA V and have ScriptHook V, and ScriptHook V Dot Net installed, but Menyoo will not open with the requirements completed.

**Comment by AGRessoR777** [comment]
remove ScriptHook V Dot Net and the problem will go away! I don't know any other options yet.

**Comment by The-UnXplained** [comment]
This official version of the menu hasn't been updated for some time now, hence why it will not work / or fully work correctly. I suggest you search for my version thats always fully up-to-date.

**Comment by jjjjj8797** [comment]
> This official version of the menu hasn't been updated for some time now, hence why it will not work / or fully work correctly. I suggest you search for my version thats always fully up-to-date.

Do you still have a working file for the 2944 version of fivem?

**Comment by The-UnXplained** [comment]
Yes, your lucky i still have the previous version! i have put the download link back on my webpage. (link to webpage on my profile)

**Comment by jjjjj8797** [comment]
> Yes, your lucky i still have the previous version! i have put the download link back on my webpage. (link to webpage on my profile)

Thanks!!!

**Comment by jwallansheel** [comment]
downloaded the latest steam edition for gta5 ,the agents of sabotage one,added menyoo,asi loader,scripthookv and dotnet . doesnt work . anybody facing this same issues?
